words/tails.txt

```diff
@@ -555,3 +555,21 @@ neurotrichini
 uropsilus
 desmana
 fynbos
+panthera
+onca
+pardus
+leo
+snow
```

Comment:
Does it count if the word isn't an animal by itself?

Reply:
As the Chief Officer of Tails and Scales ™️ 

![image](https://media.tenor.com/Q1BaZqGyv0QAAAAC/schitts-creek-alexis-rose.gif)